Pages/content.html

```diff
@@ -14,8 +34,14 @@
     <a href="../Pages/quote.html">Quote</a> |
 </nav>
 
-    <h3>Access Content files here.</h3>
-    <!- Sharing content. -->
-    <h4> abc ebook:</h4><a href="../Content/dummy.pdf" target="_blank">Dummy Pdf</a>
+<h3>Access Content files here.</h3>
+<!- Sharing content. -->
+<h4> abc ebook:</h4><a href="../Content/dummy.pdf" target="_blank">Dummy Pdf</a>
+
+<div id="footer">
+    <p> Made With ♥ By <a href="http://dreamstar.pro" target="_blank"><span>Dream Star Pro</span></a></p>
+    <p> Open Source Project <a href="https://github.com/DreamStarPro/mWebsite"
+                               target="_blank"><span>click here</span></a></p>
```

Comment:
Change color of the link  "click here" to blue

Reply:
I have fixed link color to blue.  :thumbsup:
Commit: 64c4e462401bc182a01c931fb061cb513ea401e4